geonode/qgis_server/tests/test_helpers.py

```diff
@@ -251,22 +252,21 @@ def test_delete_orphan(self):
         # register file list
         layer_path = settings.QGIS_SERVER_CONFIG['layer_directory']
         tiles_path = settings.QGIS_SERVER_CONFIG['tiles_directory']
-        geonode_layer_path = os.path.join(settings.MEDIA_ROOT, 'layers')
 
-        qgis_layer_list = set(os.listdir(layer_path))
-        tile_cache_list = set(os.listdir(tiles_path))
-        geonode_layer_list = set(os.listdir(geonode_layer_path))
+        qgis_layers = set(os.listdir(layer_path))
+        tile_caches = set(os.listdir(tiles_path))
+        _, geonode_layers = map(set, storage.listdir("layers"))
```

Comment:
I think a comment would be useful explaining why a `listdir` operations requires `map(set`.

In fact, since you're discarding the set resulting from the first list returned by `storage.listdir` why not just use something like:
```python
# Django's storage.listdir always returns a tuple of two lists, the latter of which
# contains the file names we need.
geonode_layers = set(storage.listdir("layers")[1])
```

Note the same applies below.